Header header
uint32 size
itm_nonlinear_mpc/itm_trajectory_point[] traj
float64[] data